### ISSUE TITLE
How to generate labels

### ISSUE THREAD

**Issue by kyrie2-robot-perception** [HKUST-Aerial-Robotics/SG-Reg#4]
Hello, I'm quite interested in your solid work. While when I want to construct the training set, I found the outputs of FM-Fusion do not contain four files, including xyzi.pth edges.pth nodes.csv and transform.txt. So how can we generate these files? Can some relevant scripts be offered? Thanks!

**Comment by kyrie2-robot-perception** [comment]
And also, it will be more helpful if the code to generate matches data can be further offered. Thanks!

**Comment by glennliu** [comment]
Hi @kyrie2-robot-perception 

Thanks for your interests in our work.
I have not cleaned up the training data yet. It is in the same format as the validation data, which I have uploaded to Huggingface.
To generate the match data, you can use ```sgreg/dataset/generate_gt_association.py``` to create the data association.

I will tidy-up my training dataset to include those ```*.th```. So, you can use the data easily. I will try to upload them to Huggingface this month.

Regards

**Comment by kyrie2-robot-perception** [comment]
Thanks for your help!
